xarray_jax/__init__.py

```diff
@@ -333,33 +302,12 @@ def assign_jax_coords(
   return assign_coords(x, jax_coords=jax_coords or jax_coords_kwargs)
 
 
-def wrap(value):
```

Comment:
Please could you leave a version of this as the identity function for backwards-compatibility (documented as deprecated and for backwards compat only)